src/data.js

```diff
@@ -0,0 +1,23 @@
+// eslint-disable-next-line import/prefer-default-export
```

Comment:
- [ ] Linters help us write better code. Kindly enable linters here.

  For the `import/prefer-default-export` linter issue you could do something like this:
  #### in data.js file
    ```
    const data = [
      ....
    ]
    export default data
  ```
  **In fetch.js file** where you imported `data` simply remove the curly braces since it is a default import.

  You can refer this [StackOverFlow answer](https://stackoverflow.com/a/53206122/10283280) for further elaboration.